apiary.apib

```diff
@@ -333,9 +332,6 @@ device first by calling the
 <a href="/#reference/0/organizations/unregister-license">Organizations UnregisterLicense</a>
 endpoint.
 
-* The response to Organizations RegisterLicense contains a RoleID for the
-license - if it is SDK_DEVICE, then obtaining a user token is mandatory.
```

Comment:
* The response to <a href="/#reference/0/organizations/register-license">Organizations.RegisterLicense</a> contains a RoleID for the license.

Reply:
Please make `Organization RegisterLicense` a link: https://dfxapiversion10.docs.apiary.io/#reference/0/measurements/list